src/main/java/org/folio/rest/impl/TitlesApi.java

```diff
@@ -102,7 +102,7 @@ public void putOrdersTitlesById(String id, Title entity, Map<String, String> oka
       return;
     }
 
-    titlesService.saveTitle(entity, new RequestContext(vertxContext, okapiHeaders))
+    titlesService.updateTitle(entity, new RequestContext(vertxContext, okapiHeaders))
```

Comment:
we should not create 2 similar public methods with similar meaning: save nad update, because it can be hard to clients to choose correct one, we should have only single methog

Reply:
update method name to saveTitleWithAcqUnitsCheck